requirements.txt

```diff
@@ -0,0 +1,2 @@
+pkg-resources==0.0.0
```

Comment:
please remove it because installation of dependencies crashes with error `Could not find a version that satisfies the requirement pkg-resources==0.0.0`